libs/utils/utils.js

```diff
@@ -655,17 +666,47 @@ async function loadFooter() {
   await loadBlock(footer);
 }
 
+function decorateSection(section, idx) {
+  let links = decorateLinks(section);
+  decorateDefaults(section);
+  const blocks = section.querySelectorAll(':scope > div[class]:not(.content)');
+
+  const { doNotInline } = getConfig();
+  const blockLinks = [...blocks].reduce((blkLinks, block) => {
+    const blockName = block.classList[0];
+    links.forEach((link) => {
+      if (!block.contains(link)) return;
+
+      if (link.classList.contains('fragment')
+        && MILO_BLOCKS.includes(blockName) // do not inline consumer blocks (for now)
+        && !doNotInline.includes(blockName)) {
+        link.href = `${link.href}#_inline`;
```

Comment:
not sure how much it matters, but you might already have #_inline on some inline fragment links here.